src/box/lua/net_box.lua

```diff
@@ -165,7 +165,8 @@ local function stream_wrap_index(stream_id, src)
         _src = src,
     }, {
         __index = src,
-        __serialize = stream_index_serialize
+	__serialize = stream_index_serialize,
+	__autocomplete = stream_index_serialize
```

Comment:
Bad indentation. Please don't use tabs in Lua files. Fix here and everywhere else in this patch.

Reply:
Presently, luacheck doesn't check coding style. It'd be nice if it did though. We can also make checkpatch warn about tabs in Lua files.

Reply:
I patched checkpatch to warn about tabs in all files except C source files.